packages/main/src/components/SelectDialog/index.tsx

```diff
@@ -143,6 +143,10 @@ export interface SelectDialogPropTypes
    *
    * @default ListMode.SingleSelect
    */
+  numberOfSelectedItems?: number;
+  /**
+   * This event will be fired when the value of the search field is changed by a user - e.g. at each key press
+   */
```

Comment:
This description doesn't match the prop `mode`?

Reply:
Oops 😆 
Fixed